app/services/onlinetesting/phase1/Phase1TestEvaluation.scala

```diff
@@ -18,23 +18,29 @@ package services.onlinetesting.phase1
 
 import model.EvaluationResults.{ Amber, Green, Red, Result }
 import model.SchemeType._
-import model.exchange.passmarksettings.{ PassMarkThreshold, Phase1PassMarkSettings }
+import model.exchange.passmarksettings.{ PassMarkThreshold, Phase1PassMarkSettings, Phase1PassMarkThresholds }
 import model.persisted.{ SchemeEvaluationResult, TestResult }
 
 trait Phase1TestEvaluation {
 
   def evaluateForGis(schemes: List[SchemeType], sjqTestResult: TestResult, passmark: Phase1PassMarkSettings): List[SchemeEvaluationResult] = {
-    schemes map { scheme =>
-      val schemePassmark = passmark.findPassmarkForScheme(scheme)
+    for {
```

Comment:
shall we refactor this to extract into a method to be reused with gis and non gis